chore: add first CSS lab prototype (#413)
freeCodeCamp/CurriculumExpansion: frontend-cert/css-projects/business-card/user-stories.md

1. Your page background should be `rosybrown`. The overall font should be `Arial`, with a fallback of `sans-serif`.
   
2. All of the `p` tags should have a top and bottom margin of `5px` and a left and right margin of `0px`.

> 3. The links on the page shouldn't be underlined. However, when you hover over the links, they should be underlined. 

Reviewer: ```suggestion
3. All links on the page should have a default of no underline. When you hover over the links, there should be an underline
```

4. You should have a `div` with the class `business-card`. 
   
5. The `business-card` class should be `300px` wide and it should have a background color of `lavenderblush`. There should be a padding of `20px` all around and a top margin of `100px`. The text should be center aligned and the font size should be `16px`. The left and right margins should be set to `auto`.

6. Inside the `buisness-card` div, there should be an `img` tag with the class `profile-image`. The image source should be set to `flower.jpg` and there should be an `alt` attribute should be set to `Profile Image`. The image size should be `150px by 150px`. The bottom margin should be `10px` and there should be a `2px` solid `lightgray` border around the image.

> 7. Inside the `business-card` div, you should have an `h1`, `h2, `and `h3` tag.

Reviewer: ```suggestion
7. Inside the `business-card` `div`, you should have an `h1`, `h2, `and `h3` element.
```

> 7. Inside the `business-card` `div`, after the `img` element,  you should have three `p` elements with the class `full-name`, `designation` and `company` respectively.

Reviewer 2: ```suggestion
7. Inside the `business-card` `div`, after the `img` element, you should have three `p` elements with the class `full-name`, `designation`, and `company`, respectively.
```

8. The `h1` element should contain your name. Its font size should be `24px` and the `color` should be `darkslategray`.

9. The  `h2` element should have your designation. Its `font-size` property should be `18px` and the `color` should be set to `gray`.

10. The `h3` element should have the company name. Its font size should be `16px` and the `color` should be set to `darkgray`.

> 11. There should be an `hr` tag below the `h3` element. The `hr` tag in general should have a `1px` dashed top border in `slategray` colour. There should be a top and bottom margin of `20px` and a left and right margin of `0px`.

Reviewer: ```suggestion
11. There should be an `hr` element below the `h3` element. The `hr` element in general should have a `1px` dashed top border in `slategray` color. There should be a top and bottom `margin` of `20px` and a left and right `margin` of `0px`.
```

> 12. After the `hr` element, there should be three `p` tags. In the first `p` tag, there should be an `a` tag with `href` property set to send an email to any email address. The `a` tag should have an email address as the text. In the second `p` tag, the text should be a phone number. The third `p` tag should have the text `Portfolio: ` and an `a` tag with any valid link. Add an `hr` tag after the third `p` tag.

Reviewer: we should avoid using tags but instead elements 
we should only use tags when referring to opening or closing tags.

Also, the mailto portion needs to be removed because they were not taught that so we can't ask them that.

```suggestion
12. After the `hr` element, there should be three `p` elements. In the first `p` element, there should be an `a` element with `href` property set to an email address. The `a` tag should have an email address as the text. In the second `p` element, the text should be a phone number. The third `p` element should have the text `Portfolio: ` and an `a` element with any valid link. Add an `hr` element after the third `p` element.
```
